src/components/search/SuggestionProductCard/suggestion-product-card.scss

```diff
@@ -3,6 +3,10 @@
 .suggestion-product-card {
```

Comment:
I think we missed this class, should we update to `data-suggestion-product-card` ? 👀  (If so, remember to update  on [SuggestionProductCard.tsx](https://github.com/vtex-sites/gatsby.store/blob/b06bd2c58ed940bf7998adb91f208c267c3ba7f2/src/components/search/SuggestionProductCard/SuggestionProductCard.tsx#L33))

Reply:
You're right! I've replaced the classes with `data-fs-*` [here](https://github.com/vtex-sites/gatsby.store/pull/7/commits/3802754f668a0b7f88cbcdd6e539177ccfefc92c).